contracts/optimistic-ethereum/OVM/chain/OVM_CanonicalTransactionChain.sol

```diff
@@ -141,10 +141,40 @@ contract OVM_CanonicalTransactionChain is iOVM_CanonicalTransactionChain, Lib_Ad
             uint40
         )
     {
-        (, uint40 nextQueueIndex,,) = _getBatchExtraData();
```

Comment:
This change is stylistic to match other places that I saw use no spaces when doing this kind of destructuring